packages/fast-components-react-msft/src/carousel/carousel.tsx

```diff
@@ -80,27 +89,35 @@ class Carousel extends Foundation<
     public render(): React.ReactElement<HTMLDivElement> {
         return (
             <div {...this.unhandledProps()} className={this.generateClassNames()}>
-                <Flipper
-                    direction={FlipperDirection.previous}
-                    onClick={this.handlePreviousClick}
-                    className={get(
-                        this.props,
-                        "managedClasses.carousel_flipperPrevious",
-                        ""
-                    )}
-                />
+                {this.isSingleSlide && (
+                    <Flipper
+                        direction={FlipperDirection.previous}
+                        onClick={this.handlePreviousClick}
+                        className={get(
+                            this.props,
+                            "managedClasses.carousel_flipperPrevious",
+                            ""
```

Comment:
I don't think i've seen an answer yet, but, still confused why we have `""` on classNames.  I've seen this used a lot.  What's the case "" would not be necessary or why couldn't the get method support 2 or 3 input parameters using function overloading?

Reply:
`get` is a utility function from lodash which takes 3 params. The first is an object, the second is a path, and the third is a default value. Get goes to retrieve the value of the object's path - if it's undefined we return the default value. In our scenario, if the className doesn't exist, we pass empty strings.

We use `get` here because managed classes are a prop value and not a hard coded value. They are essentially a variable which could be undefined. In this file, the class names don't actually exist. They ultimately don't exist until they are passed by the HOC (manageJSS) to this component as props. In the case this is undefined, we would rather pass an empty string (no class) rather than a non-string `undefined` value.